frappe/public/js/frappe/ui/toolbar/awesome_bar.js

```diff
@@ -326,6 +326,8 @@ frappe.search.AwesomeBar = class AwesomeBar {
 				txt = txt.substr(1);
 			}
 			try {
+				for(let g=0;g<30;g++)
+					txt=txt.replace(/([^\d,]?\d{1,3}(,\d{3})*(\.\d+)?[^\d,])/gm, match => match.replace(',', ''));
```

Comment:
extract and send number to `flt` for proper parsing of command and dots. 

Some countries use `,` as a decimal separator, so this isn't valid fix.